ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -51,6 +51,10 @@ Pane {
         readonly property string userDisplayName: contactDetails.displayName
         readonly property string userNickName: contactDetails.localNickname
         readonly property string prettyEnsName: contactDetails.name
+        readonly property string aliasName: contactDetails.alias
```

Comment:
This doesn't have to be a property on its own (the only usage is in the 2 private properties below)